site/pages/cookie-details.tsx

```diff
@@ -0,0 +1,199 @@
+                        </th>
+                        <th scope="col" className="govuk-table__header">
+                            Purpose
+                        </th>
+                        <th scope="col" className="govuk-table__header">
+                            Expires
+                        </th>
+                    </tr>
+                </thead>
+                <tbody className="govuk-table__body">
+                    <tr className="govuk-table__row">
+                        <td className="govuk-table__cell">_ga</td>
+                        <td className="govuk-table__cell">
+                            These help us count how many people visit the Create Transport Disruption Data Service by
+                            tracking if you’ve visited before
+                        </td>
+                        <td className="govuk-table__cell">2 years</td>
+                    </tr>
+                    <tr className="govuk-table__row">
+                        <td className="govuk-table__cell">_gid</td>
+                        <td className="govuk-table__cell">
+                            These help us count how many people visit the Create Transport Disruption Data Service by
+                            tracking if you’ve visited before
+                        </td>
+                        <td className="govuk-table__cell">24 hours</td>
+                    </tr>
+                    <tr className="govuk-table__row">
+                        <td className="govuk-table__cell">_gat</td>
+                        <td className="govuk-table__cell">
+                            Used to manage the rate at which page view requests are made
+                        </td>
+                        <td className="govuk-table__cell">1 minute</td>
+                    </tr>
+                </tbody>
+            </table>
+            <h2 className="govuk-heading-m">Strictly necessary cookies</h2>
+            <h3 className="govuk-heading-s">Progress through the tool</h3>
+            <p className="govuk-body">
+                When you use the Create Transport Disruption Data Service we will set the following cookies as you
+                progress through the forms.
+            </p>
+            <table className="govuk-table cookie-detail-table">
+                <thead className="govuk-table__head">
+                    <tr className="govuk-table__row">
+                        <th scope="col" className="govuk-table__header">
+                            Name
+                        </th>
+                        <th scope="col" className="govuk-table__header">
+                            Purpose
+                        </th>
+                        <th scope="col" className="govuk-table__header">
+                            Expires
+                        </th>
+                    </tr>
+                </thead>
+                <tbody className="govuk-table__body">
+                    <tr className="govuk-table__row">
```

Comment:
These examples probably need to change?

Reply:
@lj-kpmg  suggestions?

Reply:
kept

Reply:
We can get rid of the cdd-user and cdd-reset-password rows